src/lifecycle/created.js

```diff
@@ -55,7 +55,7 @@ export default function (opts) {
 
   return function () {
     let info = data(this, opts.id);
-    let isNative = this.createdCallback;
+    let isNative = !this.hasOwnProperty('_isPolyfill') && this.createdCallback || !(this._isPolyfill = true);
```

Comment:
> Yep, but if we can pre-calculate then we don't have to do it multiple times.

What I mean by that is that we also check for whether or not a definition is native in the `attached` and `detached` callbacks. If we can pre-calc this, then we can also reuse that in those callbacks.